### PR TITLE
Add structure aa1 to implement 1

### PR DESCRIPTION
**Pull request by Aichnerc**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Documentation (non-breaking change which adds documentation)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently there is no Admin page.
- Currently there is no side navigation.

**What is the new behavior (if this is a feature change)?**
- The Admin page has been added.
- A side navigation has been implemented. The menu items have been added and successfully tested.

**Other information**:
- The navigation has been adopted from a past project of mine. [https://github.com/api-link/Front](https://github.com/api-link/Front)

### CODE REVIEW

File: src/components/molecules/SideNav/index.jsx [hunk 1]
```diff
@@ -0,0 +1,145 @@
+//#region > Imports
+//> React
+// Contains all the functionality necessary to define React components
+import React from "react";
+
+//> Additional modules
+// Side Navigation
+import SideNav, { NavItem, NavIcon, NavText } from "@trendmicro/react-sidenav";
+// Side Navigation CSS
+import "@trendmicro/react-sidenav/dist/react-sidenav.css";
```
Comment (pinterid) [inline src/components/molecules/SideNav/index.jsx]: Can this be moved into the CSS section?

File: src/components/molecules/SideNav/index.jsx [hunk 2]
```diff
@@ -0,0 +1,145 @@
+//#region > Imports
+//> React
+// Contains all the functionality necessary to define React components
+import React from "react";
+
```
Comment (pinterid) [inline src/components/molecules/SideNav/index.jsx]:
```suggestion
```

File: src/components/molecules/SideNav/index.jsx [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+//> Additional modules
+// Side Navigation
+import SideNav, { NavItem, NavIcon, NavText } from "@trendmicro/react-sidenav";
+// Side Navigation CSS
+import "@trendmicro/react-sidenav/dist/react-sidenav.css";
+
```
Comment (pinterid) [inline src/components/molecules/SideNav/index.jsx]:
```suggestion
```

File: src/components/pages/AdminPage/index.jsx [hunk 4]
```diff
@@ -0,0 +1,86 @@
+//#region > Imports
+//> React
+// Contains all the functionality necessary to define React components
+import React from "react";
+
```
Comment (pinterid) [inline src/components/pages/AdminPage/index.jsx]:
```suggestion
```

File: src/components/pages/AdminPage/index.jsx [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+                  return <p>Home</p>;
+              }
+            })()}
+          </MDBContainer>
+        </div>
+      </div>
+    );
+  }
+}
+
```
Comment (pinterid) [inline src/components/pages/AdminPage/index.jsx:78]: Missing endregion

File: src/components/pages/AdminPage/index.jsx [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+export default HomePage;
```
Comment (pinterid) [inline src/components/pages/AdminPage/index.jsx:80]: Missing export region

**Review by pinterid — CHANGES_REQUESTED** [review]
A few changes have to be made

**Review by pinterid — APPROVED** [review]
Looks good

**Review by kleberbaum — APPROVED** [review]
Should be ok to merge.

**Pull request merged**